src/components/areaMenu/AposAreaMenuItem.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <button @click="click" class="apos-area-menu__button" :data-action="item.action">
+    <component 
+      v-if="item.icon"
+      :size="15"
+      class="apos-area-menu__item-icon"
+      v-bind:is="icon"
+    ></component>
+    <span>{{ item.label }}</span>
```

Comment:
The span seems superfluous, fwiw. Flex should treat the text as a flex item already.